internal/agent/flags/flags.go

```diff
@@ -124,35 +124,35 @@ func applyFileConfig(config *config.AgentConfig) {
 }
 
 func readEnvVars() {
-	if envRunAddr := os.Getenv("ADDRESS"); envRunAddr != "" {
+	if envRunAddr, exists := os.LookupEnv("ADDRESS"); exists && envRunAddr != "" {
```

Comment:
Принято такое именование:

```suggestion
	if envRunAddr, ok := os.LookupEnv("ADDRESS"); ok && envRunAddr != "" {
```